modules/style.js

```diff
@@ -23,6 +24,19 @@ function updateStyle(oldVnode, vnode) {
       elm.style[name] = cur;
     }
   }
+
+  for (name in oldStyle) {
+    if (name === 'delayed') {
+      for (name in oldStyle.delayed) {
+        if (!curHasDel || !(name in style.delayed)) {
+          elm.style[name] = '';
+        }
+      }
+    }
+    if (name !== 'remove' && name !== 'destroy' && !(name in style)) {
+      elm.style[name] = '';  // the DOM is weird eh?
```

Comment:
That's why I wondered if you considered it a bug - vs user error? On the one hand, it seems like 'style' should work like props, attrs, and h itself -- you are just describing the dom state at a point in time, and don't have to worry about imperatively removing past state. On the other, it does add to the code complexity tho and a (tiny) performance hit - so I would be ok taking it out and just documenting how to remove styles in the standard weird dom way.